File: zad2/src/main/java/com/example/zad2/ui/welcome/WelcomeViewModel.java
package com.example.zad2.ui.welcome;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

public class WelcomeViewModel extends ViewModel {

    private final MutableLiveData<String> mText;

    public WelcomeViewModel() {
        mText = new MutableLiveData<>();
        mText.setValue("Welcome to StayFit app");
    }

    public LiveData<String> getText() {
        return mText;
    }
}

File: zad4/src/main/java/com/example/zad4/ui/welcome/WelcomeViewModel.kt
package com.example.zad4.ui.welcome

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class WelcomeViewModel : ViewModel() {
    private val mText: MutableLiveData<String>
    val text: LiveData<String>
        get() = mText

    init {
        mText = MutableLiveData()
        mText.value = "Welcome to StayFit app"
    }
}